docs/dg/dev/architecture/aggregated-module-architecture.md

```diff
@@ -0,0 +1,101 @@
+
+### Feature Structure
+
+Example of the Aggregated Module structure:
+
+```text
+[FeatureName]
+├── Client
+│   └── [FeatureName]
+│       ├── Exception
+│       ├── Search
+│       ├── Plugin
+│       │   └── Elasticsearch
+│       │   └── Catalog
+│       │   └── ... 
+│       ├── [FeatureName]ClientInterface.php
+│       ├── [FeatureName]Client.php
+│       ├── [FeatureName]Config.php
+│       ├── [FeatureName]DependencyProvider.php
+│       ├── [FeatureName]Factory.php
+│       └── ...
+├── Glue
+│   └── [FeatureName] 
+│       ├── Controller
+│       │   ├── BackendApi
+│       │   │   └── [BackendApiEntityName]ResourceController.php
+│       │   └── StorefrontApi
+│       │       └── RestApi
+│       │           └── [RestApiEntityName]ResourceController.php
+│       ├── Processor
+│       │   ├── BackendApi
+│       │   │   └── [BackendApiEntityName]Reader
+│       │   └── StorefrontApi
+│       │       └── RestApi
+│       │           └── [RestApiEntityName]Reader
+│       ├── Plugin
+│       │   ├── BackendApi
+│       │   └── StorefrontApi
+│       ├── [FeatureName]Config.php
+│       ├── [FeatureName]DependencyProvider.php
+│       ├── [FeatureName]Factory.php
+│       └── ...
+├── Service
+│   └── [FeatureName]
+│       ├── [FeatureName]Config.php
+│       ├── [FeatureName]DependencyProvider.php
+│       ├── [FeatureName]Service.php
+│       ├── [FeatureName]ServiceInterface.php
+│       ├── [FeatureName]Factory.php
+│       └── ...
+├── Shared
+│   └── [FeatureName]
+│       ├── [FeatureName]Config.php
+│       ├── [FeatureName]Constants.php
+│       └── ...
+├── Yves
+│   └── [FeatureName]
+│       ├── [FeatureName]Config.php
+│       ├── [FeatureName]DependencyProvider.php
+│       ├── [FeatureName]Factory.php
+│       └── ...
+└── Zed
+   └── [FeatureName]
+       ├── Business
+       │   ├── [FeatureName]Factory.php
+       │   ├── [FeatureName]Facade.php
+       │   ├── [FeatureName]FacadeInterface.php
+       │   └── ...
+       ├── Communication
+       ├── Persistence
+       ├── Presentation 
+       ├── [FeatureName]Config.php
+       └── [FeatureName]DependencyProvider.php
+```
+
+### Feature development changes on the Project Level
+
+* The project-level [application layering](https://docs.spryker.com/docs/dg/dev/architecture/architectural-convention#application-layers) remains unchanged.
+* Feature code at the project level now uses the SprykerFeature namespace instead of Spryker.
+* In the new Aggregated Module Architecture, all sub-domains of the same feature are placed into a single feature folder, rather than being split across multiple modules.
```

Comment:
so, this is a core change only? Does the project have to do something in a different way?